files/generate_vars.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+import configparser
+import subprocess
+import shlex
+
+
+class GenerateMappingFile():
+    prefix = "[Generate Mapping File] "
+
+    def run(self):
+        print("\n%sStart Generate Varialbe Mapping File "
+              "For oVirt Ansible Disaster Recovery\n" % self.prefix)
+        site, username, password, ca, output_file = self._init_vars()
+        print("%ssite address: %s \n"
+              "%susername: %s \n"
+              "%spassword: %s \n"
+              "%sca file location: %s \n"
+              "%soutput file location: %s \n"
+              % (self.prefix,
+                 site,
+                 self.prefix,
+                 username,
+                 self.prefix,
+                 password,
+                 self.prefix,
+                 ca,
+                 self.prefix,
+                 output_file))
+
+        ansible_play = "../examples/dr_generate_var.yml"
```

Comment:
Will this link work also from rpm install? Where do you put the examples then? Currently in doc. If you actually use them, perhaps move to /usr/share/ovirt-ansible-disaster-recovery/playbooks or something.

Reply:
So what you are saying is that I will need to add a new folder containing this playbook? which will be located in playbooks?